pyccel/codegen/printing/ccode.py

```diff
@@ -2176,6 +2183,9 @@ def _print_Assign(self, expr):
         # Inhomogenous tuples are unravelled and therefore do not exist in the c printer
         if isinstance(rhs, (NumpyArray, PythonTuple)):
             return prefix_code+self.copy_NumpyArray_Data(expr)
+        if(isinstance(rhs, (CudaFull))):
+            # TODO add support for CudaFull
+            return " \n"
         if isinstance(rhs, (NumpyFull)):
             return prefix_code+self.arrayFill(expr)
         lhs = self._print(expr.lhs)
```

Comment:
Cuda concepts should not be found in this file

Reply:
https://github.com/pyccel/pyccel-cuda/pull/64/commits/96c3f292f8532f0f396bf8016af9f0f9cc6e8ea3